README_template.md

```diff
@@ -0,0 +1,23 @@
+<h1 align="center">Nushell Scripts</h1>
+<div align="center">
+	<a href="https://github.com/nushell/nu_scripts/issues">
+		<img src="https://img.shields.io/github/issues/nushell/nu_scripts?label=Issues">
+	</a>
+	<img src="https://img.shields.io/github/issues-pr/nushell/nu_scripts?color=brightgreen">
+	<img src="https://img.shields.io/github/stars/nushell/nu_scripts?color=yellow&label=Stars">
+	<img src="https://img.shields.io/badge/Scripts-{{scriptcount}}*-blue" title="Some scripts are excluded." alt="Badge that shows there are {{scriptcount}} scripts">
+</div>
+<br>
+
+This is a place to share Nushell scripts with each other. If you'd like to share your scripts, create a PR that adds it to the repo.
+
+## Running Scripts
+
+You can run Nushell scripts in a few different ways.
+
+1. You can type `nu <script name>`.
```

Comment:
it might be good to have a blurb in here about what happens if you script has a `def main [] { }` i.e. you can do nu script_name.nu --param1 --param2. last I checked we could do that with scripts that don't have main.